src/main/java/org/triplea/client/launch/screens/staging/PlayerMapping.java

```diff
@@ -0,0 +1,22 @@
+package org.triplea.client.launch.screens.staging;
+
+import java.util.Map;
+
+import games.strategy.net.INode;
+
+/**
+ * A model object to represent the mapping of which players will play as which sides.
+ * This model object can be used to send data over network, or to render GUI contents.
+ */
+public class PlayerMapping {
```

Comment:
This class appears to be unused, I'd like to hold it off until it's actually used.

Reply:
Good eye,  removed in: b5cb4e588d733842fcd65048774236a5904b10ef